app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
It is only used here. What do you think about putting the content_for back into the view, but also moving the partial out of `shared`, since it isn't actually shared at this point?